lib/lte_link_control/lte_lc.c

```diff
@@ -799,6 +799,16 @@ int lte_lc_power_off(void)
 	return 0;
 }
 
+int lte_lc_deinit(void)
+{
+	if (is_initialized) {
+		is_initialized = false;
+		return lte_lc_power_off();
+	}
```

Comment:
I think `at_notif_deregister_handler()` should also be called to clean up the AT notif handler used by this module

Reply:
ok